01-exploring-pegasus.ipynb

```diff
@@ -646,21 +646,17 @@
     "from dwave.cloud.exceptions import *\n",
     "\n",
     "try:\n",
-    "    qpu_advantage = DWaveSampler(solver={'topology__type': 'pegasus'})\n",
-    "    qpu_2000q = DWaveSampler(solver={'topology__type': 'chimera'})\n",
-    "    \n",
-    "    qpus = {'Advantage': qpu_advantage, 'DW-2000Q': qpu_2000q}\n",
-    "\n",
-    "    print(\"Connected to Advantage {} and 2000Q {}.\".format(qpu_advantage.solver.id, qpu_2000q.solver.id))\n",
+    "    qpu_advantage = DWaveSampler(solver={'topology__type': 'pegasus'})   \n",
+    "    print(\"Connected to Advantage {}.\".format(qpu_advantage.solver.id))\n",
```

Comment:
> For posterity, we could leave Chimera/2000Q numbers

Added 
![image](https://user-images.githubusercontent.com/34041130/236280836-27a14886-3df7-46d1-9898-e8b66f43e555.png)

and

![image](https://user-images.githubusercontent.com/34041130/236281419-301225ad-05ce-45d6-adbc-f9450162090a.png)

Reply:
> include Advantage2 prototype (Zephyr) here?

I accidently wrote in in [this PR](https://github.com/dwave-examples/feature-selection-notebook/pull/18/files/e8d2c09a32127f9770460ea01e9148d437dcec55#r1185092761) but that comment should be here: "Because this JN was written to introduce the new topology with the release of Advantage, I'd rather keep Zephyr out of it. In this context, a single example topology is sufficient."